releases/release-1.10/release-notes-draft.md

```diff
@@ -1,3 +1,1018 @@
+
+    * Remove deprecated ContainerVM support from GCE kube-up.  ([#58247](https://github.com/kubernetes/kubernetes/pull/58247), [@mikedanese]([https://github.com/mikedanese](https://github.com/mikedanese)))
+
+    * Remove deprecated kube-push.sh functionality.  ([#58246](https://github.com/kubernetes/kubernetes/pull/58246), [@mikedanese]([https://github.com/mikedanese](https://github.com/mikedanese)))
+
+    * Remove deprecated container-linux support in gce kube-up.sh.  ([#58098](https://github.com/kubernetes/kubernetes/pull/58098), [@mikedanese]([https://github.com/mikedanese](https://github.com/mikedanese)))
+
+    * Remove deprecated and unmaintained photon-controller kube-up.sh.  ([#58096](https://github.com/kubernetes/kubernetes/pull/58096), [@mikedanese]([https://github.com/mikedanese](https://github.com/mikedanese)))
+
+    * Remove deprecated and unmaintained libvirt-coreos kube-up.sh.  ([#58023](https://github.com/kubernetes/kubernetes/pull/58023), [@mikedanese]([https://github.com/mikedanese](https://github.com/mikedanese)))
+
+    * Remove deprecated and unmaintained windows installer.  ([#58020](https://github.com/kubernetes/kubernetes/pull/58020), [@mikedanese]([https://github.com/mikedanese](https://github.com/mikedanese)))
+
+    * Remove deprecated and unmaintained openstack-heat kube-up.sh.  ([#58021](https://github.com/kubernetes/kubernetes/pull/58021), [@mikedanese]([https://github.com/mikedanese](https://github.com/mikedanese)))
+
+    * Remove deprecated vagrant kube-up.sh. ([#58118](https://github.com/kubernetes/kubernetes/pull/58118),[@roberthbailey](https://github.com/roberthbailey))
+
+* The DaemonSet controller, its integration tests, and its e2e tests, have been updated to use the apps/v1 API. Users should, but are not yet required to, update their scripts accordingly. ([#59883](https://github.com/kubernetes/kubernetes/pull/59883), [@kow3ns]([https://github.com/kow3ns](https://github.com/kow3ns)))
+
+* MountPropagation feature is now beta. As consequence, all volume mounts in containers are now "rslave" on Linux by default. To make this default work in all Linux environments the entire mount tree should be marked as shareable, e.g. via "mount --make-rshared /". All Linux distributions that use systemd already have root directory mounted as rshared and hence they need not do anything. In Linux environments without systemd we  recommend running "mount --make-rshared /" during boot before docker is started, [@jsafrane]([https://github.com/jsafrane](https://github.com/jsafrane)))
+
+# Known Issues
+
+This section is still in flux.  Please make sure all issues are documented in the known Issue accumulator: [https://github.com/kubernetes/kubernetes/issues/59764](https://github.com/kubernetes/kubernetes/issues/59764)
+
+# Deprecations
+
```

Comment:
the etcd2 deprecation ([first announced in 1.9](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.9.md#api-machinery-2)) is significant enough that I think we should repeat it in every release until support is removed.

Suggest adding:
> `etcd2` as a backend is deprecated and support will be removed in Kubernetes 1.13

cc @kubernetes/sig-api-machinery-pr-reviews

Reply:
Added.